### DISCUSSION TITLE
Either don't update or provide corresponding Python packages

### DISCUSSION THREAD

**Discussion by defencedog** [termux/termux-packages#10598] [Ideas]
I am a chemical engineer & am deeply frustrated by termux python package with zero support of previous releases. Whenever you update python provide corresponding packages for scipy, numpy, pandas, sympy are garbadged. Literally i waste weeks searching for solutions

**Comment by ghost** [comment]
The reason why we don't keep older versions of packages is the disk space of the Termux repositories. we currently have 64GB of provided space and our infrastructure is still limited.

Most python packages and modules can be installed without problems at all with the current python version. but there are problems with specific python packages that can't be installed even with older versions of python  as you mentioned. numpy, scipy can be installed from its pointless repo

And there are issues with pandas installation from mostly 2 years ago, that's before python 3.10 is released. perhaps try with the suggestions on one of these closed issues for panda installation 
https://github.com/termux/termux-packages/issues?q=pandas

I appreciate your feedback but being rude without suggesting proper solutions or further elaboration *why it's garbaged?* is something we cannot accept solving

**Reply by defencedog** [reply]
I am sorry if I appear rude. It was not my intention.

Its-pointless builds are for python v3.9 & therefore scipy installed via that path is NOT available to pip...

The solution is that whenever termux team opts for python upgrade it can provide deb packages for well-known libraries ...at least this is what Conda, WinPython do

**Comment by xtkoba** [comment]
It is not prohibited by our packaging policy to package a Python module when there is no trivial workaround for installing it through `pip`. Please feel free to open a packaging request for such a module.

Conversely, we won't package a Python module when it is installable through pip with a trivial workaround.

**Comment by ChiefMikeK** [comment]
![Termux-Vault_01](https://user-images.githubusercontent.com/20070053/168416712-7c5431be-b1f6-4c5e-b490-fc627fc6226a.png)
There are some old packages uploaded to archive.org via narabot and others anonymously but since they are zip/tarballed they can't`¯\_(ツ)_/¯` be indexed!
-One example; https://archive.org/search.php?query=creator%3A%28termux%29&and[]=year%3A%222017%22
- I would like to see some collections created which needs 50min items per coll
- `Termux(5-6)Packages` _keep it short_ and URL`%20`friendly
- `Termux(7-_)Packages`
- https://help.archive.org/help/collections-a-basic-guide/
- we would probly need to store the SHA512 hashes on github and make a few admins for these?
- Heres a quick sample I've made for a collection icon.
